packages/docs/docs/utils-api/incremental-tree.md

```diff
@@ -16,23 +20,31 @@ Get a new tree instance.
 ```ts
 constructor(
   depth: number,
-  zeroValue: number = 0,
+  zeroValue: Node = 0,
   arity: number = 2
 ): IncrementalMerkleTree
 ```
 
+## Node
+
+The type of each leaf.
+
+```ts
+type Node = any
```

Comment:
I don't like this type, we should use `bigint` to be more clear.

Reply:
it is defined in zk-kit
https://github.com/privacy-scaling-explorations/zk-kit/blob/f9b209d6006fd454fc464bbd7e1ad3cfab6389d0/packages/incremental-merkle-tree/src/types/index.ts#L1